tests/test_astrometryTask.py

```diff
@@ -162,6 +162,17 @@ def doTest(self, pixelsToTanPixels, order=3):
         )
         self.assertLess(len(resultsRefSelect.matches), len(results.matches))
 
+        # try again, allowing magnitude outlier rejection.
+        config.doMagnitudeOutlierRejection = True
+        solverMagOutlierRejection = AstrometryTask(config=config, refObjLoader=self.refObjLoader)
+        self.exposure.setWcs(distortedWcs)
+        resultsMagOutlierRejection = solverMagOutlierRejection.run(
+            sourceCat=sourceCat,
+            exposure=self.exposure,
+        )
+        self.assertEqual(len(resultsMagOutlierRejection.matches), len(resultsRefSelect.matches))
```

Comment:
I think all that this is testing is that we get the same number of matches whether we do outlier rejection or not? That doesn't seem like a useful test of the outlier rejection system itself. Could you set a tighter outlier threshold and check that some things actually got rejected? I don't know what this test data actually looks like.

There's a `maxAngSep` and `maxPixSep` calcluation above this that could be recycled (in a closure or free function, maybe?) to check the actual result.

Reply:
So the test catalogs are all 0-magnitude, perfect.  I didn't add any scatter to this, I just wanted to test here that my code ran smoothly and didn't reject anything.  Adding scatter to the test reference catalogs would be another whole thing that I was hoping to avoid.

Reply:
Upon further review, this is necessary to do all the proper checking, and I have added some outliers to the source and reference catalogs.